docs/endpoints/post-token-refresh.md

```diff
@@ -95,10 +98,10 @@ An error response might look like the following:
 
 | Property | Data Type | Description |
 | :--- | :--- | :--- |
-| `advertising_token` | string | An encrypted advertising (UID2) token for the user. |
+| `advertising_token` | string | An encrypted UID2 token (advertising token) for the user. |
```

Comment:
I think this should be along the line of:
_also known as UID2 token, it's an encrypted form of a raw UID2 representing the user._

Reply:
Right, "UID2 token" is already encrypted

Reply:
Updated. Thx.